testsuite/integration/microprofile-tck/metrics/pom.xml

```diff
@@ -189,6 +189,97 @@
+                        <executions>
+                            <execution>
+                                <id>bootable-jar-observability-packaging</id>
+                                <goals>
+                                    <goal>package</goal>
+                                </goals>
+                                <phase>process-test-resources</phase>
+                                <configuration>
+                                    <output-file-name>test-wildfly-observability.jar</output-file-name>
+                                    <hollowJar>true</hollowJar>
+                                    <record-state>false</record-state>
+                                    <log-time>${galleon.log.time}</log-time>
+                                    <offline>true</offline>
+                                    <plugin-options>
+                                        <jboss-maven-dist/>
+                                        <jboss-fork-embedded>${galleon.fork.embedded}</jboss-fork-embedded>
+                                        <optional-packages>passive+</optional-packages>
+                                    </plugin-options>
+                                    <feature-packs>
+                                        <feature-pack>
+                                            <groupId>${testsuite.ee.galleon.pack.groupId}</groupId>
+                                            <artifactId>${testsuite.ee.galleon.pack.artifactId}</artifactId>
+                                            <version>${testsuite.ee.galleon.pack.version}</version>
+                                            <inherit-configs>false</inherit-configs>
+                                            <inherit-packages>false</inherit-packages>
+                                        </feature-pack>
+                                    </feature-packs>
+                                    <layers>
+                                        <layer>datasources-web-server</layer>
+                                        <layer>observability</layer>
+                                    </layers>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <systemPropertyVariables>
+                                <install.dir>${project.build.directory}/wildfly</install.dir>
+                                <bootable.jar>${project.build.directory}/test-wildfly-observability.jar</bootable.jar>
+                                <arquillian.xml>arquillian-bootable.xml</arquillian.xml>
+                                <module.path>${project.build.directory}/wildfly/modules</module.path>
```

Comment:
I think this is the line that you added to fix the errors that were logged before [1].
That being said, though, the bootable JAR should not depend on this property so it is quite strange that it fixed the behavior.
@jfdenise, WDYT? 
WFWIP [2] has been filed to track this issue.

[1]
https://github.com/fabiobrz/wildfly/commit/727d96332fcb6e7ae851e6be6b1858cc41ffe981

[2]
https://issues.redhat.com/browse/WFWIP-331

Reply:
@fabiobrz I am not able to reproduce locally, with or without module.path (even with a module.path pointing to an invalid location the test passes, I am calling:
 mvn clean test -Dts.bootable -DtestLogToFile=true -pl metrics/ -am -Dtest=org.eclipse.microprofile.metrics.tck.metrics.ConcurrentGaugeFunctionalTest

Reply:
It was added as I saw it in use in case of `ts.layers` profile. @jfdenise is right, it does not affect test results. Removed it in latest commit (force push): efdbd789ff1ea6f3930e1c4584643ad9c2b6e350

Reply:
@spriadka @fabiobrz , does it means that the test ConcurrentGaugeFunctionalTest fails for another reason?

Reply:
Hi @spriadka and @jfdenise. Sorry for the late reply. I've fetched the PR source branch and have successfully built it.
Thanks for looking into this. @jfdenise, I'll delete the WFWIP as soon as I check again after the PR is merged.